Lord Forte Backlog.rb

```diff
@@ -354,7 +352,7 @@ def refresh
           c = ' ' if c == "\n"
           if c =~ /[ \t]/
             c = '' if @lastc =~ /[\s\n\f]/
-            if pos[:x] + get_next_word_size(c, text) > contents.width - @right_margin - 96
+            if pos[:x] + get_next_word_size(c, text) > contents.width - @right_margin

```

Comment:
It's not necessary to subtract the width of the faceset because its width is already accounted for in the starting value of `x`.